### DISCUSSION TITLE
FanHouse

### DISCUSSION THREAD

**Discussion by DeepSeaOne** [DIGITALCRIMINALS/OnlyFans#544] [Ideas]
Any plans to add FanHouse support in the future?

**Comment by DIGITALCRIMINAL** [comment]
Yes and no... but mainly no right now.

W/ Fanhouse I'd have to use a library that parses javascript

![image](https://user-images.githubusercontent.com/89371864/182720427-41f1c19f-a8ee-43c9-9a01-43b4e0edc240.png)
